multicluster/cmd/service-mirror/main.go

```diff
@@ -57,6 +57,7 @@ func Main(args []string) {
 	enablePprof := cmd.Bool("enable-pprof", false, "Enable pprof endpoints on the admin server")
 	localMirror := cmd.Bool("local-mirror", false, "watch the local cluster for federated service members")
 	federatedServiceSelector := cmd.String("federated-service-selector", k8s.DefaultFederatedServiceSelector, "Selector (label query) for federated service members in the local cluster")
+	probeSvc := cmd.String("probe-service", "", "Name of the target cluster probe service")
```

Comment:
I think based on the helm template, this flag will always be provided.  But to be extra safe, does it make sense to use "probe-gateway-<name>" as the default value?

Reply:
Yep, good spot :+1:

Reply:
Well, actually this is expecting the full name of the service with the target cluster name as a suffix, so a partial name won't do...

Reply:
Defeated by markdown again 🤦‍♀️ 
My suggestion was supposed to include the name by markdown ate it.  I've edited my comment so that it shows up properly.

Reply:
I can't set that default directly in that line, but I can change that flag to something like `probe-service-prefix` and then append the name before passing it to `restartClusterWatcher()` . Is something like that what you have in mind?

Reply:
probably not worth it, given that our templates always set this flag.